ipa/ipa_test.go

```diff
@@ -263,6 +263,39 @@ func TestCRSGeneration(t *testing.T) {
 	}
 }
 
+func TestInsideDomainEvaluation(t *testing.T) {
```

Comment:
If you run this test in the current `main`, you'll get an error which is precisely the situation we want to resolve. Now this passes as expected.